tests/unit_tests/synthetic/libraries/cufftXt2hipfftXt.cu

```diff
@@ -56,5 +56,84 @@ int main() {
   cufftXtCallbackType_t FFT_CB_ST_REAL = CUFFT_CB_ST_REAL;
   cufftXtCallbackType_t FFT_CB_ST_REAL_DOUBLE = CUFFT_CB_ST_REAL_DOUBLE;
   cufftXtCallbackType_t FFT_CB_UNDEFINED = CUFFT_CB_UNDEFINED;
+
+  // CHECK: hipfftResult fftResult;
+  cufftResult fftResult;
+
+  // CHECK: hipfftHandle fftHandle;
+  cufftHandle fftHandle;
+
+  // CHECK: hipfftXtSubFormat fftXtSubFormat;
+  cufftXtSubFormat fftXtSubFormat;
+
+  // CHECK: hipfftXtCopyType fftXtCopyType;
+  cufftXtCopyType fftXtCopyType;
+
+  // CHECK: hipLibXtDesc *descptr;
+  // CHECK-NEXT: hipLibXtDesc *input_desc;
+  // CHECK-NEXT: hipLibXtDesc *output_desc;
+  cudaLibXtDesc *descptr;
+  cudaLibXtDesc *input_desc;
+  cudaLibXtDesc *output_desc;
+
+  // CHECK: hipLibXtDesc **desc;
+  cudaLibXtDesc **desc;
```

Comment:
It is better to initialize all the declared variables if we want to compile the test results someday.